slices/libx11-data.yaml

```diff
@@ -0,0 +1,83 @@
+package: libx11-data
+
+essential:
+  - libx11-data_copyright
+
+slices:
+  data:
+    essential:
+      - libx11-data_locale
+    contents:
+      /usr/share/X11/XErrorDB:
+
+  locale:
```

Comment:
More of a slice design question for the maintainers:
For packages like this, should we embed all locale files in the main slice for the package (as proposed here), or should the main package slice only include as few locales as necessary to have a working system (`C`, `en_US.UTF-8`?) and then have a separate slice with all locales

Reply:
Very good question @HadrienPatte . Your line of questioning is in the right place.

The short answer is: the latter :)

The guidelines are:
 - if you are slicing a package for general purpose, then you can create more generic slice names (like `bins`, `libs`, etc.) with a more comprehensive list of contents, based on their nature.
 - however, if you have a specific use case, the recommendation is that you create the most minimalistic slice possible, delivering just the minimum necessary to make your use case work

It is completely ok to have both, since the same path can appear many times within the slice definitions file, and slices are virtually unlimited.